### PR TITLE
Fix bad URLs in HandEyeCalibration readme

### PR DESCRIPTION
**Pull request by chrisasc**
Fixed these two issues:
Absolute URLs in readme - https://github.com/zivid/zivid-cpp-samples/issues/116
Bad links in readme - https://github.com/zivid/zivid-cpp-samples/issues/133

### CODE REVIEW

**Comment by chrisasc** [comment]
I was not able to link _ZividHandEyeCalibration_ to a local file, the CLI executable file we have in Zivid/bin. So I linked it to link to our KB article (https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/96469274/CLI+tool+for+Hand-Eye+calibration) instead. I think this makes more sense as we dont have that .exe in Github.

**Comment by chrisasc** [comment]
I checked and the links works for me, could you double check that you are on the right branch when checking the links. When you click on an PR, GitHub sometimes changes your branch back to the master brach:)

**Comment by SatjaSivcev** [comment]
Added @knatten to the review since he has reported the issue.

File: source/Applications/Advanced/HandEyeCalibration/README.md [hunk 1]
```diff
@@ -13,26 +13,27 @@ To fully understand Hand-Eye Calibration, please see the [tutorial](https://zivi
    4. Repeat 1.-3. until 10-20 pose pairs are collected
    5. Enter command to perform calibration and return a **Transformation Matrix**
 
-[ZividHandEyeCalibration](C:\Program Files\Zivid\bin\ZividHandEyeCalibration.exe): (no source)
+[ZividHandEyeCalibration](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/96469274)
```
Comment (SatjaSivcev) [inline source/Applications/Advanced/HandEyeCalibration/README.md:16]: why not use the same standard here as elsewhere? I mean use ZividHandEyeCalibration-url here and then at the bottom of the file add the address.

File: source/Applications/Advanced/HandEyeCalibration/README.md [hunk 2]
```diff
@@ -13,26 +13,27 @@ To fully understand Hand-Eye Calibration, please see the [tutorial](https://zivi
...
 * Zivid primarily operate with a (4x4) Transformation Matrix (Rotation Matrix + Translation Vector). This example shows how to use Eigen to convert to and from:
   * AxisAngle, Rotation Vector, Roll-Pitch-Yaw, Quaternion
 
-[SampleHandEyeCalibration-url]: https://www.zivid.com/hubfs/softwarefiles/releases/1.6.0+7a245bbe-26/doc/cpp/zivid_sample_code.html#autotoc_md10
-[UtilizeEyeInHandCalibration-url]: https://github.com/zivid/zivid-cpp-samples/blob/master/Applications/Advanced/HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp
-[PoseConversions-url]: https://github.com/zivid/zivid-cpp-samples/blob/master/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp
\ No newline at end of file
+[HandEyeCalibration-url]: HandEyeCalibration/HandEyeCalibration.cpp
+[UtilizeEyeInHandCalibration-url]: UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp
+[ZividHandEyeCalibration-url]: file:///C:/Program&#32;Files/Zivid/bin/ZividHandEyeCalibration.exe
```
Comment (SatjaSivcev) [inline source/Applications/Advanced/HandEyeCalibration/README.md:38]: This seems weird. What is file:///C:/Program&#32?

Comment (knatten) [reply source/Applications/Advanced/HandEyeCalibration/README.md:38]: Yeah this looks wrong. The variable is never user either.

Comment (chrisasc) [reply source/Applications/Advanced/HandEyeCalibration/README.md]: Read online that I needed to add "&#32;" where a link normally has space. But im not using that link anyway, so I removed it

File: source/Applications/Advanced/HandEyeCalibration/README.md (hunk 1 above)
Comment (chrisasc) [reply source/Applications/Advanced/HandEyeCalibration/README.md]: Done, standardized all links now

**Comment by chrisasc** [comment]
Fixed changes, ready for next round

**Comment by knatten** [comment]
In zivid-sdk, we try to avoid force pushing during review. Instead we add fixup commits, and then squash and rebase when the review is done. This makes it much easier for reviewers to keep track of what changed since last time they reviewed, and if they have the branch checked out locally they won't have to force reset it. I'm just a guest in this repository, but I suggest you adopt the same workflow. As a reviewer I would certainly appreciate it. What do you think?

File: source/Applications/Advanced/HandEyeCalibration/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # Hand Eye Calibration
 
-To fully understand Hand-Eye Calibration, please see the [tutorial](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/72450049) in our Knowledge Base.
+To fully understand Hand-Eye Calibration, please see the [tutorial][tutorial-url] in our Knowledge Base.
```
Comment (SatjaSivcev) [inline source/Applications/Advanced/HandEyeCalibration/README.md:3]: Make t capital T -> Tutorial-url

**Review by SatjaSivcev — CHANGES_REQUESTED** [review]
Just one minor change needed

**Comment by chrisasc** [comment]
> Just one minor change needed

Done, ready for next round

**Comment by SatjaSivcev** [comment]
Looks good to me. You can rebase and merge.

**Review by SatjaSivcev — APPROVED** [review]

**Pull request merged**